Frontend/src/index.ui.tsx

```diff
@@ -7,10 +7,27 @@ import { createWorkloadClient, InitParams } from '@ms-fabric/workload-client';
 
 import { fabricLightTheme } from "./theme";
 import { App } from "./App";
+import { convertGetItemResultToWorkloadItem } from "./utils";
+import { callItemGet } from "./controller/SampleWorkloadController";
+import { ItemPayload } from "./models/SampleWorkloadModel";
+import { DataWithId } from './models/SampleWorkloadModel';
 
 export async function initialize(params: InitParams) {
     const workloadClient = createWorkloadClient();
-
+    workloadClient.action.onAction(async function ({ action, data }) {
+        switch (action) {
+            case 'sample.tab.onInit':
+                const { id } = data as DataWithId;
+                const getItemResult = await callItemGet(
+                    id,
+                    workloadClient
+                );
+                const item = convertGetItemResultToWorkloadItem<ItemPayload>(getItemResult);
```

Comment:
How will this behave in case of error?

E.G if for some reason the item was not able to load?

Reply:
added try and catch for item loading

Reply:
+ added basic handlers for all tab actions